From e35eb4084807437f372444e03885667f75598907 Mon Sep 17 00:00:00 2001
From: Roman Kovtyukh <HelloDearGrandma@gmail.com>
Date: Fri, 13 Jun 2025 00:57:41 +0100
Subject: [PATCH] Upgrade minimum required cmake to 3.15

In CMake 4.0
> Compatibility with CMake < 3.5 has been removed from CMake.

https://cmake.org/cmake/help/latest/release/4.0.html#deprecated-and-removed-features
---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 57cdba8..b07d850 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,7 +1,7 @@
 # Evolution-EteSync build script
 
-cmake_minimum_required(VERSION 3.1)
-cmake_policy(VERSION 3.1)
+cmake_minimum_required(VERSION 3.15)
+cmake_policy(VERSION 3.15)
 
 project(evolution-etesync
 	VERSION 1.1.2
